blocks/article-body-block/_index.scss

```diff
@@ -0,0 +1,148 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-article-body {
+	> * {
+		@include scss.block-components("article-body-children");
+		@include scss.block-properties("article-body-children");
+	}
+
+	a {
```

Comment:
Shouldn't these already be `<Link />`s that can be styled with the components nesting?

Reply:
ah yes... makes sense.